### DISCUSSION TITLE
I have school ID with valid date, but my application for Student Developer Pack is not approved

### DISCUSSION THREAD

**Discussion by bvastdacc** [community/community#52413] [GitHub Education]
Hi all,

I want to register for a Student Developer Pack of GitHub Education. I have registered with a school-issued email, but I have to upload documents that prove my current student status. Then, I uploaded a photo of my school ID with valid date like picture below:
![image](https://user-images.githubusercontent.com/129304430/230811219-0c333b9b-429b-4116-8c80-c5fc74835261.png)

And I got rejected by below reason:
![image](https://user-images.githubusercontent.com/129304430/230811387-06c36eb9-4f27-49f4-b361-77d5ce10bd7e.png)

I uploaded a photo of my transcript, a photo of school portal with student status on it, etc. and I got rejected by the same reason. Please help me, I don't have much documents to prove student status and I'm really frustrated after being rejected many times like this.

**Comment by samus-aran** [comment]
Hi @bvastdacc 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
